app/src/js/components/admin_dashboard.tsx

```diff
@@ -51,7 +51,7 @@ const drawerWidth = 285;
     views: Array<Object>
 }*/
 
-const styles = (theme) => ({
+const styles = (theme:any) => ({
```

Comment:
```suggestion
const styles = (theme: any) => ({
```

Maybe consider install tslint so that it can capture/fix all these style issues?